routes/admin/users.rb

```diff
@@ -62,6 +62,9 @@
     tool_authorization = params[:tool_authorization]
     expiration_date = params[:expiration_date]
     expiration_date_operation = params[:expiration_date_operation]
+    hidden_sort_by_name = params[:hidden_sort_by_name]
```

Comment:
Don't need to have hidden in these parameter names

Reply:
done